addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -179,16 +179,19 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
                 }
                 const parsedInput = event.detail.buffer && parse.float(event.detail.buffer) || 0;
                 if(lastId != selectedLine.cid)
-                    this._showDecreaseQuantityPopup();
+                    await this._showDecreaseQuantityPopup();
                 else if(currentQuantity < parsedInput)
                     this._setValue(event.detail.buffer);
                 else if(parsedInput < currentQuantity)
-                    this._showDecreaseQuantityPopup();
+                    await this._showDecreaseQuantityPopup();
             } else {
                 let { buffer } = event.detail;
                 let val = buffer === null ? 'remove' : buffer;
                 this._setValue(val);
             }
+            if (this.env.pos.config.iface_customer_facing_display) {
+                this.env.pos.send_current_order_to_customer_facing_display();
+            }
```

Comment:
There is a call of this statement in `_setValue`, you can remove it.

Reply:
True, but I don't think it is in _showDecreaseQuantityPopup(), when removing this statement and decreasing value of the product to 0, customer display does not get updated.

Reply:
I meant this: https://github.com/odoo/odoo/blob/06817417e190d64ab6b0fc6151e02b6090b1fd39/addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js#L209
I think it can be removed.